FreeRTOS/FreeRTOS-Plus-TCP
# Add unit tests for IPv6 Sockets and Code coverage

Description
-----------
Implement unit tests for sockets IPv6 and 100% code coverage for unit tests.

Test Steps
-----------
Unit tests build and executed. LCOV coverage tested.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
